react/react1/week1/todo-app/src/components/Todos.js

```diff
@@ -0,0 +1,30 @@
+const todoItem = [
+  {
+    id: 1,
+    activity: 'Get out of bed',
+    dateline: 'Wed Sep 13 2017',
+  },
+  {
+    id: 2,
+    activity: 'Brush teeth',
+    dateline: 'Thu Sep 14 2017',
+  },
+  {
+    id: 3,
+    activity: 'Eat breakfast',
+    dateline: 'Fri Sep 15 2017',
+  },
+];
+
+export function Todos() {
+  return (
+    <ul>
+      {todoItem.map((item) => (
```

Comment:
It would be nice if you created a component for each item of the Array. =)

Reply:
I have a difficulty to make put in separate component, but I will try to practise 🙂 Thank you 🙏